actions/checkout_desktop/action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Setup for all'
+description: "Setup shared for all jobs"
+runs:
+  using: 'composite'
+  steps:
+      - name: Checkout Desktop
+        uses: actions/checkout@v4
+        with:
+          repository: 'oxen-io/session-desktop'
+          path: 'desktop'
+          ref: 'standardised_strings_qa_2'
```

Comment:
Should this `ref` be removed?

Reply:
we can't for the same reason. Also, the validation tools  for desktop are only on that branch for now, so the only way to run them is by checking out that branch